### PR TITLE
added MFA warnings

### PR DESCRIPTION
**Pull request by dbradish-microsoft**

### CODE REVIEW

File: docs-ref-conceptual/whats-new-overview.md
Comment (dcaro) [inline docs-ref-conceptual/whats-new-overview.md:18]: Can we explicitly call out what is in the blog post that service principals and managed identities auth based will not be impacted?

**Comment by dbradish-microsoft** [comment]
@dcaro , do you want to give this a second review?

**Review by dcaro — APPROVED** [review]
LGTM

**Pull request merged**
